BOLT/getting_started/world_program.mdx

```diff
@@ -26,70 +26,56 @@ Initiating a project with bolt init automatically generates a simple usage examp
 Create a new world instance as demonstrated below:
 
 ```typescript
-const registry = await Registry.fromAccountAddress(
-  provider.connection,
-  registryPda
-);
-const worldId = new anchor.BN(registry.worlds);
-const worldPda = FindWorldPda(new anchor.BN(worldId));
-const initializeWorldIx = createInitializeNewWorldInstruction({
-  world: worldPda,
-  registry: registryPda,
+const initializeNewWorld = await InitializeNewWorld({
   payer: provider.wallet.publicKey,
+  connection: provider.connection,
 });
-
-const tx = new anchor.web3.Transaction().add(initializeWorldIx);
-const txSign = await provider.sendAndConfirm(tx);
+const signature = await provider.sendAndConfirm(initializeNewWorld.transaction);
+const worldPda = initializeNewWorld.worldPda; // we can use this later
 ```
 
 ## Adding a New Entity
 
 To add a new entity:
 
 ```typescript
-const world = await World.fromAccountAddress(provider.connection, worldPda);
-const entityId = new anchor.BN(world.entities);
-entityPda = FindEntityPda(worldId, entityId);
-
-let createEntityIx = createAddEntityInstruction({
-  world: worldPda,
+const addEntity = await AddEntity({
   payer: provider.wallet.publicKey,
-  entity: entityPda,
+  world: worldPda,
+  connection: provider.connection,
 });
-const tx = new anchor.web3.Transaction().add(createEntityIx);
-await provider.sendAndConfirm(tx);
+const signature = await provider.sendAndConfirm(addEntity.transaction);
+const entityPda = addEntity.entityPda; // we can use this later
 ```
 
 ## Attaching Components to an Entity
 
 For attaching components:
 
 ```typescript
-const positionComponentPda = FindComponentPda(
-  positionComponent.programId,
-  entityPda,
-  ""
-);
-let initComponentIx = createInitializeComponentInstruction({
+const inititializeComponent = await InitializeComponent({
   payer: provider.wallet.publicKey,
   entity: entityPda,
-  data: positionComponentPda,
-  componentProgram: positionComponent.programId,
+  componentId: positionComponent.programId,
+  seed: "my_id", // In case an entity has multiple of the same component, we differenciate
 });
-const tx = new anchor.web3.Transaction().add(initComponentIx);
-await provider.sendAndConfirm(tx);
+const signature = await provider.sendAndConfirm(
+  inititializeComponent.transaction
+);
+const componentPda = transaction.componentPda; // we can use this later
 ```
 
 ## Applying Systems
 
 To apply a system:
 
 ```typescript
-let applySystemIx = createApplyInstruction({
-  componentProgram: positionComponent.programId,
-  boltSystem: systemMovement.programId,
-  boltComponent: positionComponentPda,
+const applySystem = await ApplySystem({
+  authority: provider.wallet.publicKey,
+  system: systemMovement.programId,
+  entity: entityPda,
+  components: [positionComponent.programId],
+  seeds: ["my_id"],
```

Comment:
For the "simple" first example, I would avoid setting the seeds (or we can add an extra note). It might be unfamiliar to some game developers.

Reply:
Fair enough 👌 done!